x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -483,36 +495,41 @@ export const visualizationTypes: VisualizationType[] = [
     fullLabel: i18n.translate(
       'xpack.lens.xyVisualization.stackedPercentageBarHorizontalFullLabel',
       {
-        defaultMessage: 'Horizontal percentage bar',
+        defaultMessage: 'Bar horizontal percentage',
       }
     ),
+    groupLabel: groupLabelForBar,
   },
   {
     id: 'area',
     icon: LensIconChartArea,
     label: i18n.translate('xpack.lens.xyVisualization.areaLabel', {
       defaultMessage: 'Area',
     }),
+    groupLabel: groupLabelForLineAndArea,
   },
   {
     id: 'area_stacked',
     icon: LensIconChartAreaStacked,
     label: i18n.translate('xpack.lens.xyVisualization.stackedAreaLabel', {
       defaultMessage: 'Stacked area',
```

Comment:
To be consistent with naming and match the designs.

```suggestion
      defaultMessage: 'Area stacked',
```